docker/build-tools/Dockerfile

```diff
@@ -432,3 +430,72 @@ RUN chmod 777 /go && \
 WORKDIR /
 
 ENTRYPOINT ["/usr/local/bin/docker-entrypoint"]
+
+##############
+# Clang+LLVM
+##############
+
+FROM ubuntu:xenial as clang_context
+
+RUN apt-get update && apt-get install -y xz-utils
+
+ENV LLVM_VERSION=9.0.0
+ENV LLVM_TARGET=x86_64-linux-gnu-ubuntu-16.04
+
+ENV LLVM_BASE_URL=http://releases.llvm.org/${LLVM_VERSION}
+ENV LLVM_ARCHIVE=clang+llvm-${LLVM_VERSION}-${LLVM_TARGET}
+ENV LLVM_URL=${LLVM_BASE_URL}/${LLVM_ARCHIVE}.tar.xz
+ENV LLVM_DIRECTORY=/usr/lib/llvm
+
+ADD ${LLVM_URL} /tmp
+RUN tar -xJf /tmp/${LLVM_ARCHIVE}.tar.xz -C /tmp
+RUN mkdir -p ${LLVM_DIRECTORY}
+RUN mv /tmp/${LLVM_ARCHIVE}/* ${LLVM_DIRECTORY}/
+
+###########
+# Bazel
+###########
+
+FROM ubuntu:xenial as bazel_context
```

Comment:
bionic to match the rest of the file? (unless that PR never merged - it was sent today )

Reply:
Done.